src/resources/postcss/components/_navigation.pcss

```diff
@@ -0,0 +1,93 @@
+/* = Navigation
+============================ */
+.tribe-events {
+
+	.tribe-common-c-nav {
+		display: flex;
+		justify-content: space-between;
+		border-top: 1px solid var(--color-border-default);
+		padding-top: var(--spacer-6);
+	}
+
+	.tribe-common-c-nav--prev,
+	.tribe-common-c-nav--next {
+		display: flex;
+		align-items: center;
+		color: var(--color-text-secondary);
+		font-size: var(--font-size-1);
+		font-weight: var(--font-weight-bold);
+
+		@media (--viewport-medium) {
+			font-size: var(--font-size-2)
+		}
+
+		&:hover {
+			color: rgba(var(--color-text-primary), 0.8);
+		}
+
+		&:disabled {
+			color: var(--color-text-disabled);
+		}
+	}
+
+	.tribe-common-c-nav--prev {
+
+		&:before {
+			background-image: svg-inline(caret-left);
+			background-repeat: no-repeat;
+			background-size: contain;
+			content: '';
+			height: 14px;
+			margin-right: 10px;
+			width: 12px;
+		}
+
+		&:active,
+		&:hover,
+		&:focus {
+
+			&:before {
+				background-image: svg-inline(caret-left-active);
+			}
+		}
+
+		&:disabled {
+
+			&:before {
+				background-image: svg-inline(caret-left-disabled);
+			}
+		}
+
+	}
+
+	.tribe-common-c-nav--next {
+
+		&:after {
```

Comment:
should be `&:before`

Reply:
```suggestion
		&:before {
```

Reply:
@paulmskim Is this resolved by recent commits?

Reply:
@juanfra this still needs to be fixed here.

Reply:
I'm using :before for the prev and :after for the next arrow. Is there any reason for them to live both in the same pseudo selector?

Reply:
Oh I understand, no that's fine. I know I said to move the common style to https://github.com/moderntribe/tribe-common/pull/1011/files#diff-e1567768e26c49c75604027039283f1fR38, but let's move them back to https://github.com/moderntribe/tribe-common/pull/1011/files#diff-e1567768e26c49c75604027039283f1fR50 and https://github.com/moderntribe/tribe-common/pull/1011/files#diff-e1567768e26c49c75604027039283f1fR76, the reason being there are unused styles in `.tribe-common-c-nav__prev:after` and `.tribe-common-c-nav__next:before` in the common styles.